motionplan/constraint.go

```diff
@@ -233,7 +233,7 @@ func (c *ConstraintHandler) SegmentConstraints() []string {
 }
 
 func createAllCollisionConstraints(
-	movingRobotGeometries, staticRobotGeometries, worldGeometries []spatial.Geometry,
+	movingRobotGeometries, staticRobotGeometries, worldGeometries, interactionSpaces []spatial.Geometry,
```

Comment:
I'm on the fence about whether or not we should reintroduce this verbiage.  I feel like it is more descriptive but we call it bounding regions elsewhere now, so probably should echo that here

Reply:
will update naming